src/main/java/uk/gov/ida/verifyserviceprovider/configuration/VerifyServiceProviderConfiguration.java

```diff
@@ -98,4 +100,10 @@ public Duration getClockSkew() {
     public Optional<EuropeanIdentityConfiguration> getEuropeanIdentity() {
         return europeanIdentity;
     }
+
+    @ValidationMethod(message = "eIDAS and MSA support cannot be enabled together. The VSP's eIDAS support is only support when it operates without the MSA")
+    @JsonIgnore
+    public boolean isNotMsaAndEidas() {
+        return !(msaMetadata.isPresent() && europeanIdentity.isPresent());
```

Comment:
what if `europeanIdentity` is present but set to `enabled: false`? Then the error message is confusing...

Reply:
The configuration shouldn't be there at all if you're doing matching - setting `europeanIdentity.enabled` to `false` won't do anything.

Reply:
in that case I would be inclined to update the error message to be clear, that the configuration should not be present. Otherwise it might confuse people if they somehow have it there but `enabled: false` and getting error message it cannot be enabled (which from their point of view is not)

Reply:
Fair enough, I'll change the message to say 'set' rather than 'enabled'